src/schema/objects/suffixes.yaml

```diff
@@ -502,6 +502,14 @@ VFA:
     Depending on the provided metadata fields and the sequence type,
     data may be eligible for DESPOT1, DESPOT2 and their variants
     ([Deoni et al. 2005](https://doi.org/10.1002/mrm.20314)).
+act:
+  value: act
+  display_name: Provenance activity records
```

Comment:
Let's simplify in "Activity" to be consistent with the definitions at the top of the spec: https://bids-specification--2099.org.readthedocs.build/en/2099/modality-agnostic-files/provenance.html#definitions